lib/cc/analyzer/issue_location_format_validation.rb

```diff
@@ -30,13 +30,13 @@ def valid_positions?(positions)
       def valid_position?(position)
         position &&
           (
-            (position["line"] && position["column"]) ||
-            position["offset"]
+            [position["line"], position["column"]].all? { |value| value.is_a?(Integer) } ||
+            position["offset"] && position["offset"].is_a?(Integer)
```

Comment:
I think our spec is pretty clear that these values should be numbers: I would not consider an engine sending us anything else as compliant, personally, and there's always bad edge cases with casting. I'd like to know which engines would be impacted by this change & action fixing them before shipping this, but IMHO this is a positive change.